supplementary/alica_tracing/include/tracing/TraceFactory.h

```diff
@@ -23,12 +23,16 @@ class TraceFactory : public alica::IAlicaTraceFactory
     TraceFactory&& operator=(TraceFactory&&) = delete;
 
     ~TraceFactory();
-    std::unique_ptr<alica::IAlicaTrace> create(const std::string& opName, std::optional<const std::string> parent = std::nullopt) const;
-
+    std::unique_ptr<alica::IAlicaTrace> create(const std::string& opName, std::optional<const std::string> parent = std::nullopt) const override;
```

Comment:
oos: const inside optional is a bit weird here, since the optional is pass by value.

Reply:
yeah, can change in a different pr since its a public facing api.